src/signals/incident-management/components/RadioInput/index.js

```diff
@@ -21,35 +34,33 @@ const RadioInput = ({ name, display, values }) => {
     }
 
     return (
-      <div className="radio-input">
+      <Wrapper>
         <div className="mode_input text rij_verplicht">
           {display && <Label htmlFor={`form${name}`}>{display}</Label>}
 
-          <div className="radio-input__control invoer">
+          <div className="invoer">
             {values?.map(({ key, value }) => (
-              <div className="antwoord" key={`${name}-${key}`}>
+              <ValueContainer key={`${name}-${key}`}>
                 <input
                   id={`${name}-${key}`}
                   data-testid={`${name}-${key}`}
                   className="kenmerkradio"
                   {...handler('radio', key)}
                 />
                 <label htmlFor={`${name}-${key}`}>{value}</label>
-              </div>
+              </ValueContainer>
             ))}
 
-            <p>
-              {info ? (
+            {info && (
+              <p>
                 <Info>
                   {label}: {info}
                 </Info>
-              ) : (
-                <br />
-              )}
-            </p>
+              </p>
+            )}
```

Comment:
This change will cause the elements below the list of radio buttons to jump when there is no info to be shown

Reply:
Were will the buttons jump? I have tested it and is work as expected.

Reply:
Likely related to SIG-2615. Probably not an issue that content below the radio buttons shifts down when a radio button is selected that has info text below it.